core/src/main/resources/jenkins/model/Jenkins/login_fr.properties

```diff
@@ -20,8 +20,9 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-User=Utilisateur
+signIn=S''identifier
+signUp=<a href="signup">Cr\u00e9ez un compte</a> si vous n''\u00eates pas encore membre.
+Invalid\ username\ or\ password=Nom d''utilisateur ou mot de passe incorrect
+Username=Utilisateur
```

Comment:
This even means that the current login page in french is not showing `Utilisateur` but `Username`.. so Thanks!